src/disjoint_set.jl

```diff
@@ -109,15 +143,15 @@ end
     intmap::Dict{T,Int}
     internal::IntDisjointSets
 
-    function (::Type{DisjointSets{T}}){T}(xs)    # xs must be iterable
+    function (::Type{DisjointSets{T}}){T}(xs, store_size::Bool=false)    # xs must be iterable
```

Comment:
Sorry, I missed this!  Yes, you can write a constructor so that `DisjointSets{Int}([1, 2, 3])` still works. Add

```julia
(::Type{DisjointSets{T}}){T}(xs) = DisjointSets{T, Void}(xs)
```

as an external constructor for `DisjointSets`.